src/interpolation.jl

```diff
@@ -11,37 +11,57 @@ time ``x(t)``; or a lunar core Euler angle as a function of time ``\theta_c(t)``
 
 - `t0::T`: Start time.
 - `t::AbstractVector{T}`: Vector of time instances when the timespan of the ``i``-th element of `x` ends and the ``(i+1)``-th element of `x` starts being valid. 
-- `x::AbstractArray{Taylor1{U},N}`: Vector of Taylor polynomials that interpolate the dependent variable as a function of the independent variable.
+- `x::AbstractArray{Taylor1{U},N}`: Array of Taylor polynomials that interpolate the dependent variable as a function of the independent variable.
```

Comment:
Minor comment: I'm not sure if we require `AbstractArray{Taylor1{U},N}` with respect to the `N`, or `AbstractVector{Taylor1{U}}` (N=1) is more than enough...

Reply:
All `TaylorInterpolant` produced by `PlanetaryEphemeris.jl` have `N = 2`, so we could change `AbstractArray` for `AbstractMatrix`. I do not imagine a scenario where we could need an array of a dimension other than 2. Any thoughts on this @PerezHz?

Reply:
The case `N=1` corresponds to a dense representation of a Taylor integration of a single variable wrt time, whereas `N=2` corresponds to the representation of many variables, so in that sense that's why I have only thought about cases `N=1` and `N=2`. In a related note, it could also make sense to re-structure the `TaylorInterpolant` with `N=2` case as an array of `TaylorInterpolant` with `N=1`, but probably that's a topic for another PR. So, to answer the question at hand, yeah, I think we have to cover cases `N=1` and `N=2`.

Reply:
In that case, I think that as far as this PR is concerned, we should keep `x` as an `AbstractArray` to cover cases `N=1` and `N=2`. Maybe in a future PR we could address the issue you mention.